### ISSUE TITLE
Failure selection not preserved after refresh the details view

### ISSUE THREAD

**Issue by smoralesd** [Microsoft/accessibility-insights-web#642]
## Describe the bug

On details view, automated checks, failure selection is not preserved after the user refresh the details view.

**To Reproduce**

1. Go to a web page with a11y failures, here's an [example page](https://github.com/Microsoft/accessibility-insights-web/blob/master/src/tests/end-to-end/test-resources/all.html)
2. Open FastPass -> automated checks
3. Wait for the scan to finish
4. Uncheck at least one failure/group from the failures list
5. Observe the visualization is updated properly on the target page, i.e. the visualization disappear base on what you uncheck on the failures list.
6. Refresh the details view (not the target page)
7. Observe: the failure list revert the selection (now every failure is selected)

## CodePen repro example

No CodePen but an example page on this repo: [here](https://github.com/Microsoft/accessibility-insights-web/blob/master/src/tests/end-to-end/test-resources/all.html)

## Expected behavior

User selection on the failure list should be respected, even after the user refresh the details view.

## Screenshots

![02 - selection not preserved](https://user-images.githubusercontent.com/2837582/57041295-c1eb1480-6c16-11e9-9ccc-298266fbd5ac.gif)

## Extension (please complete the following information)

- **OS**: Windows 10
- **Version**: 2.2.1, insiders 2.3.0
- **Browser Version**: Chrome 73.0.3683.103 (Official Build) (64-bit)
- Target Page [https://github.com/Microsoft/accessibility-insights-web/blob/master/src/tests/end-to-end/test-resources/all.html](https://github.com/Microsoft/accessibility-insights-web/blob/master/src/tests/end-to-end/test-resources/all.html)

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

No

## Additional context

N/A

**Comment by ferBonnin** [comment]
@smoralesd this is known and given no user feedback, won't fix.
